.github/workflows/tests-ubuntu.yml

```diff
@@ -58,11 +58,6 @@ jobs:
           env:
             TOXENV: botocore
 
-        # keep until uvloop supports 3.12
-        - python-version: "3.11"
-          env:
-            TOXENV: asyncio
```

Comment:
Shouldn’t we change it to 3.12 and remove the comment (and empty line before) rather than removing the whole thing?

Reply:
There is already a 3.12 asyncio job in the main list.

Reply:
:facepalm: